source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRButtonExpandable.xib

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="14490.70" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
+    <device id="retina4_7" orientation="portrait">
+        <adaptation id="fullscreen"/>
```

Comment:
Curious - how does the xib approach work again? Do they optionally get to pull this in to their source/project or something like that?

Reply:
This XIB will be part of iOS SDK. Devs will need to modify this XIB to have "chevrons" for showcard buttons. I've provided such changes in the XIB as part of PR #4221 for reference.